front/src/Phaser/Game/GameScene.ts

```diff
@@ -132,34 +134,30 @@ export class GameScene extends ResizableScene implements CenterListener {
         y: -1000
     }
 
-    private PositionNextScene: Array<Array<{ key: string, hash: string }>> = new Array<Array<{ key: string, hash: string }>>();
     private presentationModeSprite!: Sprite;
     private chatModeSprite!: Sprite;
     private gameMap!: GameMap;
     private actionableItems: Map<number, ActionableItem> = new Map<number, ActionableItem>();
     // The item that can be selected by pressing the space key.
     private outlinedItem: ActionableItem|null = null;
     private userInputManager!: UserInputManager;
+    private startLayerName!: string | null;
 
-    static createFromUrl(room: Room, mapUrlFile: string, gameSceneKey: string|null = null): GameScene {
+    static createFromUrl(room: Room, mapUrlFile: string): GameScene {
         // We use the map URL as a key
-        if (gameSceneKey === null) {
-            gameSceneKey = mapUrlFile;
-        }
-        return new GameScene(room, mapUrlFile, gameSceneKey);
+        return new GameScene(room, mapUrlFile);
     }
```

Comment:
The createFromUrl method now does nothing at all expect call "new".

We should completely remove it.

Reply:
done